tests/test_domain_py.py

```diff
@@ -398,21 +394,12 @@ def test_pymethod_options(app):
     assert 'Class.meth4' in domain.objects
     assert domain.objects['Class.meth4'] == ('index', 'method')
 
-    # :property:
-    assert_node(doctree[1][1][8], addnodes.index,
-                entries=[('single', 'meth5() (Class property)', 'Class.meth5', '', None)])
```

Comment:
This demonstrates what I mean: Why are there parentheses after `meth5`? `instance.meth5` wouldn’t be callable! Of course you could’ve added an exception to not render the parentheses, there’s a perfectly good `PyAttribute` that doesn’t render those parentheses.